src/hclust.jl

```diff
@@ -326,6 +457,8 @@ function hclust{T<:Real}(d::Symmetric{T}, method::Symbol)
         h = hclust2(d, slicemaximum)
     elseif method == :average
         h = hclust2(d, slicemean)
+    elseif (method == :ward1) || (method == :ward2)
```

Comment:
Have you thought about the best names? IIUC, `ward1` was the only option in R for a long time, but it's said not to implement the algorithm actually proposed by Ward. So should it be called `ward.wrong` (joking, but you see the point)? Is there any reason to prefer one algorithm over the other one? R package agnes uses `ward2` by default, and calls it `ward`. We should probably check what other software does.

Reply:
According to https://link.springer.com/article/10.1007/s00357-014-9161-z Statistica, Systat, and SPSS use ward1, whereas Matlab, SAS, and JMP use ward2. R have changed their names to "ward.D" and "ward.D2". The above paper uses "Ward1" and "Ward2", so I thought it was reasonable to stick with that.

Reply:
OK. But since Ward 2 is the actual method proposed by Ward, probably better use it by default? I wonder whether there are possible implications when it's used in combination with PCA, which is also based on variance.